shared/lib-metrics/src/main/java/gov/va/vro/metricslogging/MetricLoggerService.java

```diff
@@ -19,14 +17,15 @@
 @Conditional(NonLocalEnvironmentCondition.class)
 public class MetricLoggerService implements IMetricLoggerService {
 
-  @Value("${bip.env}")
-  public String env;
-
   private static final String APP_PREFIX = "vro_bip";
   private static final String SERVICE_TAG = "service:vro-svc-bip-api";
 
   private final MetricsApi metricsApi;
 
+  public MetricLoggerService() {
+    this.metricsApi = new MetricsApi();
```

Comment:
New MetricsApi here initializes it without the ApiClient parameter, is that okay?

Reply:
this constructor isn't needed in the first place. oops. removed in  818887e.